lesson_9/server/handlers/handlers.go

```diff
@@ -0,0 +1,25 @@
+package handlers
+
+import (
+	computer "hillel_go_cource/lesson_9/models"
+
+	"github.com/labstack/echo/v4"
+)
+
+type Handlers struct {
+	Pc *computer.Computer
+}
+
+
+func NewHandlers(pc *computer.Computer) *Handlers {
+	return &Handlers{
+		Pc: pc,
+	}
+}
+
+func (h Handlers) RegisterRouts(e *echo.Echo) {
+	e.GET("/", h.GetPcHandler)
```

Comment:
Pls, check REST best practice (i added link to tg chat), path naming - incorrect (-3)